src/core/tests/preprocess.cpp

```diff
@@ -194,6 +194,7 @@ TEST(pre_post_process, tensor_element_type_and_scale) {
     EXPECT_EQ(f->get_parameters().front()->get_element_type(), element::f32);
     EXPECT_EQ(f->get_output_element_type(0), element::i8);
     EXPECT_EQ(f->get_parameters().front()->get_layout(), Layout());
+    EXPECT_EQ(ov::layout::get_layout(f->get_parameters().front()), Layout());
```

Comment:
Suggest to add tests for 'set_layout' direct call and also for error cases (when Output<Node> is not related to Parameter or Result)